backend/src/v2/metadata/client.go

```diff
@@ -484,21 +497,22 @@ func (c *Client) PublishExecution(ctx context.Context, execution *Execution, out
 
 // metadata keys
 const (
-	keyDisplayName       = "display_name"
-	keyTaskName          = "task_name"
-	keyImage             = "image"
-	keyPodName           = "pod_name"
-	keyPodUID            = "pod_uid"
-	keyNamespace         = "namespace"
-	keyResourceName      = "resource_name"
-	keyPipelineRoot      = "pipeline_root"
-	keyCacheFingerPrint  = "cache_fingerprint"
-	keyCachedExecutionID = "cached_execution_id"
-	keyInputs            = "inputs"
-	keyOutputs           = "outputs"
-	keyParentDagID       = "parent_dag_id" // Parent DAG Execution ID.
-	keyIterationIndex    = "iteration_index"
-	keyIterationCount    = "iteration_count"
+	keyDisplayName        = "display_name"
+	keyTaskName           = "task_name"
+	keyImage              = "image"
+	keyPodName            = "pod_name"
+	keyPodUID             = "pod_uid"
+	keyNamespace          = "namespace"
+	keyResourceName       = "resource_name"
+	keyPipelineRoot       = "pipeline_root"
+	keySessionInfoDetails = "bucket_session_info"
```

Comment:
nil: Is there a reason to name it `keySessionInfoDetails` instead of `keyBucketSessionInfo` like the other naming convention? Just thinking it will be easier to debug if we are using similar names.

Reply:
good point, to take this a bit further I'm thinking this should instead be "provider_session_info", to keep it more agnostic, in case we add support beyond buckets and object stores

Reply:
addressed, I went with `store_session_info` info, as `provider_session_info` was also feeling a bit ambiguous

the fields should match the naming convention as per your comment